modules/installation-gcp-permissions.adoc

```diff
@@ -24,9 +24,7 @@ account requires the following permissions. If you deploy your cluster into an e
 
 .Required roles for the installation program
 * Compute Admin
-* IAM Security Admin
 * Service Account Admin
-* Service Account User
```

Comment:
Sorry, but I don't remember why these 2 roles are removed, could you clarify? Thanks! 
- `roles/iam.securityAdmin`
- `roles/iam.serviceAccountUser`

Reply:
I believe that service account user is covered when a user has service account admin. The Security Admin was also covered by the generic Compute Admin

Reply:
@barbacbd any updates? thanks!

Reply:
FYI The roles / permissions we used during the QE testing of [Installing a cluster on GCP into a shared VPC](https://issues.redhat.com/browse/CORS-2368)
  
_The required roles in **the service project** (i.e. the same as a normal IPI installation):_
- roles/compute.admin
- roles/compute.loadBalancerAdmin
- roles/dns.admin
- roles/iam.roleViewer
- roles/iam.securityAdmin
- roles/iam.serviceAccountAdmin
- roles/iam.serviceAccountKeyAdmin
- roles/iam.serviceAccountUser
- roles/storage.admin

_The required roles in **the host project**:_
Scenario A: Don't want the installation program to automatically create firewall rules for your cluster
- projects/openshift-qe-shared-vpc/roles/dns.networks.bindPrivateDNSZone
- roles/compute.networkUser

Scenario B: Want the installation program to automatically create firewall rules for your cluster
- projects/openshift-qe-shared-vpc/roles/dns.networks.bindPrivateDNSZone
- roles/compute.networkAdmin
- roles/compute.securityAdmin

Reply:
> @barbacbd any updates? thanks!

@jianli-wei thanks for this feedback. Brent is out on paternity leave. I will either try to push your changes into Brent's branch or clone this PR and incorporate your changes.